driver/src/price_finding/naive_solver.rs

```diff
@@ -212,109 +212,126 @@ fn order_with_buffer_for_fee(order: &Order, fee: &Option<Fee>) -> Order {
 #[cfg(test)]
 pub mod tests {
     use super::*;
+    use crate::util::{u256_to_u128};
```

Comment:
This is where the Travis is failing!

```suggestion
    use crate::util::u256_to_u128;
```